web-components/src/components/cards/PostCard/PostCard.tsx

```diff
@@ -163,33 +148,30 @@ export default function PostCard({
                 position: 'relative',
               })}
             >
-              {hasImageBlock && isPrivate && (
-                <PrivateBadge hasImageBlock={hasImageBlock} />
+              {hasImageBlock && privacyLabel && (
+                <PrivateBadge
+                  hasImageBlock={hasImageBlock}
+                  label={privacyLabel}
+                />
               )}
-              {isPrivate && !isAdmin ? (
-                <Box sx={{ backgroundColor: 'black', height: '100%' }}>
-                  Post is Private
-                </Box>
-              ) : geojson && isGISFile ? (
-                <StaticMap geojson={geojson} mapboxToken={mapboxToken} />
-              ) : (
-                imgSrc && (
+              {imgSrc && (
+                <>
                   <Image
                     className={classes.image}
                     src={imgSrc}
                     alt={imgSrc}
```

Comment:
good point, I don't think it's purely decorative, probably we could use the post title as alt? thoughts @erikalogie?

Reply:
I think it depends on the caption because the `alt` text should describe the image itself while the caption usually adds context or additional info. They should complement each other. 
I wouldn't use the post title as it could end up being rendered twice if the image fails to load, and screen readers will read it twice.

Reply:
I think it's fine to leave the alt text blank in this case as a generic description that would work for all user generated content would not add any value imho.